ui.apps/src/main/content/jcr_root/apps/aemtraining/components/structure/gridpage/body.html

```diff
@@ -1 +1 @@
-<sly data-sly-resource="${@ resourceType='aemtraining/components/content/mainpage', selectors='include'}"></sly>
\ No newline at end of file
+<sly data-sly-unwrap="${!wcmmode.edit}" data-sly-resource="${@ resourceType='aemtraining/components/content/mainpage', selectors='include'}"></sly>
```

Comment:
data-sly-unwrap doesn't work in sly tags
sly tags are not rendered in the final markup